source/develop/release-management/features/external-network-provider.html.md

```diff
@@ -0,0 +1,456 @@
+---
+title: External Network Providers
+category: feature
+authors: mmirecki
+wiki_category: Feature
+wiki_title: Features/External Network Providers
+wiki_revision_count: 1
+wiki_last_updated: 2016-01-25
+feature_name: External Network Providers
+feature_modules: engine,vdsm
+feature_status: In Development
+---
+
+# External Network Providers
+
+
+## Summary
+
+Currently it is possible to integrate oVirt with Openstack Neutron (using  the
+"Openstack Neutron Integration" feature). The purpose of this feature is to
+simplify this and to make it easier to add support for networks supplied by
+external providers. The providers will be able to provide networks using a
+simplified Neutron-like API.
+
+
+### Owner
+
+*   Feature Owner: Marcin Mirecki: [ mmirecki](User:mmirecki)
+
+<!-- -->
+
+*   Email: <mmirecki@redhat.com>
+
+### Benefit to oVirt
+
+oVirt VMs will be able to use networks definded by external providers.
+oVirt will provide an API for the external providers (vendors Software Defined
+Networking products) allowing them to easily integrate with oVirt.
+
+
+## Detailed Description
+
+Many organizations use centralized network management systems to handle all
+their networking, and would like them to manage the network topology of
+their oVirt environments. 
+Currently the only option of using external networks in oVirt is to used the
+OpenStack Neutron integration, which allows the import of OpenStack Neutron
+networks and the provisioning of VM's connected to these networks. 
+This feature is however very specific to OpenStack Neutron. 
+We would like to extend this functionality to other external network providers,
+by making it simpler, more general and less dependant on OpenStack Neutron features. 
+The result should be an API over which oVirt can communicate with external
+network management systems, and use the networks defined in them in provisioned VMs.
+
+For information about OpenStack Neutron integration please refer to:
+[OSN Integration](http://www.ovirt.org/develop/release-management/features/network/osn-integration/)
+[Detailed OSN Integration](http://www.ovirt.org/develop/release-management/features/network/detailed-osn-integration/)
+
+### Differences to OpenStack Neutron Integration
+
+*   hiding of UI features related to OpenStack Neutron, such as tenants and OpenStack Neutron driver details
+![](external_network_provider_import_dialog_changes.png "fig:external_network_provider_import_dialog_changes.png")
```

Comment:
we should make sure of not dropping formerly-released functionality inadvertently. We want to make it easy to add a generic network provider, but if someone got used to the specifics of OpenStack networking - we should let them select "OpenStack-specific options" and expose the detailed options.